bitcraze_crazyflie_2/cf2/cf2_collision_0.obj

```diff
@@ -0,0 +1,74 @@
+v 0.034218110 -0.031184591 0.007464440
```

Comment:
Make sure all OBJ files end with an empty newline (looks like some of them don't).